openid-sharedsignals-framework-1_0.md

```diff
@@ -1874,6 +1874,9 @@ publish an SLA for verification event transmission times. Event Receivers MUST N
 depend on the verification event being transmitted synchronously or in any
 particular order relative to the current queue of events.
 
+Upon receiving the verify event, the Reciver should respond with `202 Accepted` to
```

Comment:
This would only make sense if the Receiver had a push-based stream set up. There is no response code for poll-based streams. 

For push-based streams, a 202 is already required by the [push spec](https://datatracker.ietf.org/doc/html/rfc8935#name-success-response), assuming there is nothing wrong with the verify event.